apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -42,7 +43,7 @@ export interface GenericIndexTablePresenterProps<T extends IBaseViewModel> {
     tableData: T[];
     cellRenderersDefinition: CellRenderersDefinition<T>;
     heading: string;
-    filterableProperties: (keyof T)[];
+    filterableProperties: (keyof T | 'allColumns')[];
```

Comment:
I wouldn't necessarily have the user pass this in. I'd always have an 'allColumns' option, as long as there is more than 1 filterable property specified.